src/transformers/pipelines.py

```diff
@@ -821,6 +827,8 @@ def __call__(
                 Whether or not to include the decoded texts in the outputs.
             clean_up_tokenization_spaces (:obj:`bool`, `optional`, defaults to :obj:`False`):
                 Whether or not to clean up the potential extra spaces in the text output.
+            prefix (:obj:`str`, defaults to :obj:`None`):
```

Comment:
```suggestion
            prefix (:obj:`str`, `optional`):
```
(Nit: no need to mention the defaults to None for something that is optional.)